src/spring-mongo/src/main/java/net/conology/restjsonpath/JsonPathCompilerPass.java

```diff
@@ -20,18 +20,33 @@ public List<PropertyFilterNode> getQueries(JsonPathMongoParser parser) {
     }
 
     private List<PropertyFilterNode> transformRestQueries(
-        JsonPathMongoParser.RestQueriesContext ctx
-    ) {
+            JsonPathMongoParser.RestQueriesContext ctx) {
```

Comment:
do not format lines you didn't touch. also don't add trailing parenthesis that close and reopen